packages/parser/lib/Tokens.re

```diff
@@ -1,31 +1,3 @@
-[@deriving show({with_path: false})]
-type token =
-  | EOF
-  | IDENT(string) // <ident-token>
-  | BAD_IDENT // TODO: this is needed?
-  | FUNCTION(string) // <function-token>
-  | AT_KEYWORD(string) // <at-keyword-token>
-  | HASH(string, [ | `ID | `UNRESTRICTED]) // <hash-token>
-  | STRING(string) // <string-token>
-  | URL(string) // <url-token>
-  | BAD_URL // <bad-url-token>
-  | DELIM(string) // <delim-token>
-  | NUMBER(float) // <number-token>
-  | PERCENTAGE(float) // <percentage-token>
-  | DIMENSION(float, string) // <dimension-token>
-  | WS // <whitespace-token>
-  | COLON // <colon-token>
-  | SEMI_COLON // <semicolon-token>
-  | COMMA // <comma-token>
-  | LEFT_BRACKET // <[-token>
-  | RIGHT_BRACKET // <]-token>
-  | LEFT_PAREN // <(-token>
-  | RIGHT_PAREN // <)-token>
-  | LEFT_BRACE // <{-token>
-  | RIGHT_BRACE; // <}-token>
```

Comment:
Are we losing the comments?

Reply:
I don't mind because they're self describing